lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -217,7 +218,21 @@ module.exports = {
   },
 
   validateAlbAuth(auth) {
-    auth.onUnauthenticatedRequest = auth.allowUnauthenticated ? 'allow' : 'deny';
+    const hasAllowUnauthenticated = auth.allowUnauthenticated !== undefined;
+    const hasOnUnauthenticatedRequest = auth.onUnauthenticatedRequest !== undefined;
```

Comment:
This is a string property, so technically natural check would be `Boolean(auth.onUnauthenticatedRequest)`

Reply:
happy to change see comment https://github.com/serverless/serverless/pull/7780/files#r441055404